examples/hessian/single_task/input.json

```diff
@@ -0,0 +1,103 @@
+{
+  "_comment": "that's all",
+  "model": {
+    "type_map": [
+      "C",
+      "H",
+      "N",
+      "O"
+    ],
+    "descriptor": {
+      "type": "dpa2",
```

Comment:
the configuration of dpa2 looks outdated. please update with `example/water/dpa2`. 
I would recommend the `small` dpa2 configuration for a fast testing.

Reply:
Thanks for reminding. It has been updated.